Server/functions/tournaments/mappool/customBeatmapToNodesu.ts

```diff
@@ -0,0 +1,72 @@
+import { Beatmap } from "nodesu";
+import { MappoolMap } from "../../../../Models/tournaments/mappools/mappoolMap";
+
+interface CustomBeatmapToNodesu {
+    customBeatmap: NonNullable<MappoolMap["customBeatmap"]>;
+    customMappers: NonNullable<MappoolMap["customMappers"]>;
+    createdAt: NonNullable<MappoolMap["createdAt"]>;
+    lastUpdate: NonNullable<MappoolMap["lastUpdate"]>;
+}
```

Comment:
```suggestion
type CustomBeatmapToNodesu = Required<Pick<MappoolMap, "customBeatmap" | "customMappers" | "createdAt" | "lastUpdate">>;
```

Reply:
![image](https://github.com/Corsace/corsace/assets/18523454/c13fb04e-831e-4592-ba72-e32ad3631911)

Reply:
hmm i suppose `Required` doesn't cover null. forget about it then